src/models/culture.js

```diff
@@ -0,0 +1,37 @@
+export default (sequelize, DataTypes) => {
+  const Culture = sequelize.define("Culture", {
+    types: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    festivals: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    dressing: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    language: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    gallery: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    tribe: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  });
+
+  Culture.associate = (models) => {
+    Culture.belongsTo(models.Country, {
```

Comment:
youdo not need associations in culture since country already has an association that impacts culture... remove it